src/components/frame/Extensions/layouts/ColonyLayout.tsx

```diff
@@ -94,7 +94,7 @@ const ColonyLayout: FC<PropsWithChildren> = ({ children }) => {
         txButton={<TxButton />}
         userHub={<UserHubButton />}
         className={clsx(
-          'modal-blur-navigation [.show-header-in-modal_&]:z-userNavModal',
+          'modal-blur-navigation [.show-header-in-modal_&]:absolute [.show-header-in-modal_&]:right-6 [.show-header-in-modal_&]:top-8 [.show-header-in-modal_&]:z-userNavModal',
```

Comment:
This is not great in terms of readability, if you have any suggestions how to make it better please share!